packages/components/src/internal/components/user/CreateUsersModal.tsx

```diff
@@ -109,6 +112,11 @@ export class CreateUsersModal extends React.Component<Props, State> {
                     value={emailText || ''}
                     onChange={this.handleEmail}
                 />
+                {userLimitSettings?.userLimit && (
+                    <div className="create-users-limit-message">
+                        Number of users that can be added: {userLimitSettings.remainingUsers}
```

Comment:
Should this be "added or reactivated"?

Reply:
nope, this modal is just for creation. note: this matches the text for the add/create users page in LKS.